src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -209,6 +279,192 @@ private UserTypeTypeParametersFormatDefinition()
         }
         #endregion
 
+        #region User Members - Fields 
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.FieldName)]
+        [Name(ClassificationTypeNames.FieldName)]
+        [Order(After = PredefinedClassificationTypeNames.Identifier)]
+        [Order(After = PredefinedClassificationTypeNames.Keyword)]
+        [Order(Before = ClassificationTypeNames.StaticSymbol)]
+        [UserVisible(true)]
+        [ExcludeFromCodeCoverage]
+        private class UserMembersFieldNameFormatDefinition : ClassificationFormatDefinition
+        {
+            private UserMembersFieldNameFormatDefinition()
+            {
+                this.DisplayName = EditorFeaturesResources.User_Members_Fields;
+            }
+        }
+        #endregion
+        #region User Members - Enum Members 
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.EnumMemberName)]
+        [Name(ClassificationTypeNames.EnumMemberName)]
+        [Order(After = PredefinedClassificationTypeNames.Identifier)]
+        [Order(After = PredefinedClassificationTypeNames.Keyword)]
```

Comment:
why does an enum member need to be ordered after a keyword... when does that happen? (whatever the reason, please add comment).